explorer-ui/src/components/ProfileView/ProfileView.jsx

```diff
@@ -36,6 +36,7 @@ export default function ProfileView({
   let monthList = months.list;
   const handleSubmit = (event) => {
     event.preventDefault();
+    setProfile({});
```

Comment:
hmm if there is an error -> then profile will be blank D:

Reply:
fixed